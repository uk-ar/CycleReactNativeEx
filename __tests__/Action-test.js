const { Action, genActions2 } = require('../Action');

const {
  View,
  //Animated
} = require('react-native');

import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

jest
  .disableAutomock()

describe('Action tests', () => {
  beforeEach(() => {
    //jest.resetModules();
  });

  it('renders 1 CloseableView component', () => {
    const tree = renderer.create(<Action/>);
    expect(tree.toJSON()).toMatchSnapshot();
  });
  /* it('renders children when passed in', () => {
   *   const tree = renderer.create(
   *     <Action>
   *       <View key="foo"/>
   *     </CloseableView>);
   *   expect(tree.toJSON()).toMatchSnapshot();
   * });*/
});
