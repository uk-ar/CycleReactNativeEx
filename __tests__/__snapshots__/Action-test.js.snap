exports[`Action tests renders 1 CloseableView component 1`] = `
<View
  style={undefined}>
  <Text
    accessible={true}
    allowFontScaling={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 20,
        },
        Object {},
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
      ]
    }>
    ?
  </Text>
  <View
    style={
      Object {
        "width": 5,
      }
    } />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignSelf": "center",
      }
    }>
    
  </Text>
</View>
`;
